File: zl002 - MVVMInOC/MVVMInOC/Person.m
//
//  Person.m
//  MVVMInOC
//
//  Created by huang on 2021/1/16.
//

#import "Person.h"

@implementation Person

-(instancetype)init{
    if ([super init]) {
        self.firstName = @"Kobe";
        self.lastName = @"Bryant";
        self.score = 33643;
    }
    return self;
}

-(void)newScore:(int)score{
    self.score = score;
    NSLog(@"%d",self.score);
}
@end

File: zl001 - MVVMInSwiftUI/MVVMInSwiftUI/person.swift
//
//  person.swift
//  TryYiTry
//
//  Created by huang on 2021/1/16.
//

import Foundation

struct Person {
    var firstName:String
    var LasetName:String
    var score: Int
    
    mutating func newScore(score: Int) {
        self.score += score
        print(self.score)
    }
    
}
